integration_tests/test_household_structure.py

```diff
@@ -90,3 +96,52 @@ def test_households_only_have_one_reference_person(tracked_live_populations):
         ]
 
         assert len(household_ids) == len(household_ids.unique())
+
+
+def test_housing_type_does_not_change(simulants_on_adjacent_timesteps):
+    """Household types should not change for a given household"""
+    for before, after in simulants_on_adjacent_timesteps:
+        common_households = set(before["household_id"]).intersection(
+            set(after["household_id"])
+        )
+        before = (
+            before.loc[
+                before["household_id"].isin(common_households),
+                ["household_id", "household_details.housing_type"],
+            ]
+            .drop_duplicates()
+            .sort_values("household_id")
+            .set_index("household_id")
+        )
+        after = (
+            after.loc[
+                after["household_id"].isin(common_households),
+                ["household_id", "household_details.housing_type"],
+            ]
+            .drop_duplicates()
+            .sort_values("household_id")
+            .set_index("household_id")
+        )
+
+        pd.testing.assert_frame_equal(before, after)
+        assert not after.index.duplicated().any()
+
+
+def test_initialized_state_complete_coverage(populations, sim):
+    """Initialized states should include all locations from artifact"""
+    initialized_pop = populations[0]
```

Comment:
Is there a reason this test can't be on every population instead of only the first?

Reply:
I don't see why not.